ops/jenkins/bfd-cbc-build/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG VERSION_JAVA
+ARG MAVEN_VERSION
+
+FROM maven:${MAVEN_VERSION}-openjdk-${VERSION_JAVA} AS base
+
+# NOTE: necessary duplicate entry
+ARG VERSION_JAVA
+
+COPY jdk${VERSION_JAVA}.toolchains.xml /root/.m2/toolchains.xml
+
+ARG ANSIBLE_VERSION
+
+RUN apt -y update && \
+    apt -y install python3-pip jq && \
+    pip3 install ansible==${ANSIBLE_VERSION} awscli boto
+
+FROM base as tfenv
+ARG TFENV_REPO_HASH
+# NOTE: versions represented as string; Dockerfile's RUN contexts don't use arrays
+ARG TFENV_VERSIONS
+COPY tfenv-install.sh /root/tfenv-install.sh
+RUN chmod +x /root/tfenv-install.sh && \
+    /root/tfenv-install.sh "${TFENV_REPO_HASH}" "${TFENV_VERSIONS}"
```

Comment:
anyway to verify the install? verify keys or something?

Reply:
I had wondered about this myself. Any thoughts on something low-cost and meaningful?

Reply:
This was the issue I had with tfenv previously and why I personally gravitated to tfswitch. Neither are present in mainline package managers (other than homebrew), but at least with tfswitch it's a single binary so a bit easier to maintain. That said, I don't personally want to fork/manage either tool, so this is fine for now.